site/content/ncf-builder.md

```diff
@@ -21,20 +21,26 @@ Change ownership of your tree to allow ncf-api to edit it:
 The current version needs to be patched to remove built-in authentication and to change default path. 
 This won't be necessary in future versions.
 
-Edit /usr/share/ncf/api/ncf_api_flask_app/views.py to remove built-in authentication:
+
+Edit `/usr/share/ncf/api/ncf_api_flask_app/views.py` to remove built-in authentication:
```

Comment:
We changed the auth process ... and we should update the doc (it's now changing flag use_rudder_auth to False)

I'll make another PR